### PR TITLE
8.15: update docs

### PR DESCRIPTION
**Pull request by obltmachine**
Merge as soon as  branch is created and the GitHub checks are green.

### CODE REVIEW

**Comment by kyungeunni** [comment]
Might need to retrigger the job to include the latest change in https://github.com/elastic/apm-server/pull/13554, if not, I will manually update the entry in `changelogs/8.15.asciidoc`

**Comment by carsonip** [comment]
> Might need to retrigger the job to include the latest change in https://github.com/elastic/apm-server/pull/13554, if not, I will manually update the entry in changelogs/8.15.asciidoc

Haven't tried to do that. I think pushing to this PR may be easier.

**Comment by inge4pres** [comment]
> Haven't tried to do that. I think pushing to this PR may be easier.

Pushed one commit to add the entry

File: changelogs/8.15.asciidoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[[apm-release-notes-8.15]]
+== APM version 8.15
+* <<apm-release-notes-8.15.0>>
+
+[float]
+[[apm-release-notes-8.15.0]]
+=== APM version 8.15.0
+
+https://github.com/elastic/apm-server/compare/v8.14.1\...v8.15.0[View commits]
```
Comment (carsonip) [inline changelogs/8.15.asciidoc:9]:
```suggestion
https://github.com/elastic/apm-server/compare/v8.14.2\...v8.15.0[View commits]
```

The link will be broken for a while, but 8.14.2 is the version before 8.15.0. This also needs to be fixed in the main PR.

Comment (inge4pres) [reply changelogs/8.15.asciidoc:9]: sorry, now I got it 🙏🏼

**Review by carsonip — APPROVED** [review]

**Pull request merged**